enterprise_subsidy/apps/fulfillment/api.py

```diff
@@ -99,11 +99,6 @@ def _get_enterprise_customer_data(self, transaction):
         # Check if data is already cached
         cached_response = request_cache(namespace=REQUEST_CACHE_NAMESPACE).get_cached_response(cache_key)
         if cached_response.is_found:
-            logger.info(
-                'subsidy_record cache hit '
-                f'enterprise_customer_uuid={self._get_enterprise_customer_uuid(transaction)}, '
-                f'subsidy_uuid={transaction.uuid}'
-            )
```

Comment:
Our caching is quite stable now, we don't need this for debugging purposes any more.